### PR TITLE
Add text2phones handler

### PR DESCRIPTION
**Pull request by ryangawei**
Add a `Text2PhoneHandler` that hosts a service to provide text2phone function using `http://hostname/get_phones`.

Example usage:

```python
import requests

URL = "http://localhost:5000/get_phones"

data = {'text': "How are you bro", "lex_path": "lexicon/librispeech-lexicon.txt"}
r = requests.post(url=URL, json=data)
json = r.json()

print(json)
```

### CODE REVIEW

File: backend/NEU-tts-2022/audio_handler.py [hunk 1]
```diff
@@ -32,3 +33,17 @@ def AudioHandler():
...
+
+@app.route('/get_phones', methods=['POST'])
+def Text2PhoneHandler():
+    kwargs = {}
+    request_dict = json.loads(request.get_data())
+    print(request_dict)
+    kwargs['text'] = request_dict.get('text')
+    kwargs['lex_path'] = request_dict.get('lex_path')
+
+    result = text2phones.get_phones(**kwargs)
```
Comment (kevintab95) [inline backend/NEU-tts-2022/audio_handler.py:46]: Hm, if arguments are optional here, I'm wondering if calling the method without arguments would work? Not sure.

**Review by kevintab95 — COMMENTED** [review]
Thanks @ryangawei! Just one question. LGTM otherwise.

File: backend/NEU-tts-2022/audio_handler.py (hunk 1 above)
Comment (ryangawei) [reply backend/NEU-tts-2022/audio_handler.py:46]: I think `lex_path` should be optional while `text` isn't, I'll change the code to reflect this. If `text` isn't present the `get` method will raise an error. Does it make sense?

Comment (ryangawei) [reply backend/NEU-tts-2022/audio_handler.py:46]: Oh `get` won't raise error if the key isn't present...I'll manually raise the error

Comment (ryangawei) [reply backend/NEU-tts-2022/audio_handler.py:50]: Updated in L44

Comment (kevintab95) [reply backend/NEU-tts-2022/audio_handler.py:50]: Ah, I see. Thanks for explaining!

**Review by kevintab95 — APPROVED** [review]
LGTM, thanks @ryangawei!

**Pull request merged**